spring-cloud-stream/src/main/java/org/springframework/cloud/stream/converter/MessageConverterUtils.java

```diff
@@ -0,0 +1,135 @@
+/*
+ * Copyright 2015 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.cloud.stream.converter;
+
+import static org.springframework.util.MimeType.valueOf;
+import static org.springframework.util.MimeTypeUtils.APPLICATION_JSON;
+import static org.springframework.util.MimeTypeUtils.APPLICATION_OCTET_STREAM;
+
+import org.springframework.cloud.stream.tuple.Tuple;
+import org.springframework.util.ClassUtils;
+import org.springframework.util.MimeType;
+import org.springframework.cloud.stream.tuple.DefaultTuple;
+import org.springframework.util.StringUtils;
+
+
+/**
+ * Message conversion utility methods.
+ *
+ * @author David Turanski
+ * @author Gary Russell
+ */
+public class MessageConverterUtils {
+
+	/**
+	 * An MimeType specifying a {@link Tuple}.
+	 */
+	public static final MimeType X_SPRING_TUPLE = MimeType.valueOf("application/x-spring-tuple");
+
+	/**
+	 * An MimeType for specifying a String.
+	 */
+	public static final MimeType X_SPRING_STRING = MimeType.valueOf("application/x-spring-string");
```

Comment:
I think these are both obsolete and should be removed. `application/x-xd-tuple` is current